docs/log_analytics.md

```diff
@@ -0,0 +1,128 @@
+[Http Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md), and processing those logs with a [Grok Prepper](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-prepper/README.md) before ingesting them into OpenSearch through the [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md).
+
+Here is all of the components for log analytics with FluentBit, Data Prepper, and OpenSearch:
+<br />
+<br />
+![Log Analytics Components](images/LogAnalyticsComponents.png)
+<br />
+<br />
+
+In your application environment you will have to run FluentBit.
+FluentBit can be containerized through Kubernetes, Docker, or Amazon ECS.
+It can also be run as an agent on EC2.
+You should configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper.
+You will then have to deploy Data Prepper as an intermediate component and configure it to send
+the enriched log data to your OpenSearch cluster or Amazon OpenSearch Service domain. From there, you can
+use OpenSearch Dashboards to perform more intensive visualization and analysis.
+
+## Log Analytics Pipeline
+
+Log analytic pipelines in Data Prepper are extremely customizable. A simple pipeline is shown below.
+
+![](images/Log_Ingestion_FluentBit_DataPrepper_OpenSearch.jpg)
+
+## Http Source
+
+The [Http Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md) accepts log data from FluentBit. 
+More specifically, this source accepts log data in a JSON array format. 
+This source supports industry-standard encryption in the form of TLS/HTTPS and HTTP basic authentication.
+
+## Preppers
+
+The Data Prepper 1.2 release will come with a [Grok Prepper](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-prepper/README.md).
+The Grok Prepper can be an invaluable tool to structure and extract important fields from your logs in order to make them more queryable.
+
+The Grok Prepper comes with a wide variety of [default patterns](https://github.com/thekrakken/java-grok/blob/master/src/main/resources/patterns/patterns) that match against common log formats like apcahe logs or syslogs, 
+but can easily accept any custom patterns that cater to your specific log format.
+
+There are a lot of complex Grok features that will not be discussed here, so please read the documentation if you are interested.
+
+## OpenSearch sink
+
+We have a generic sink that writes the data to OpenSearch as the destination. The [opensearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md) has configuration options related to an OpenSearch cluster like endpoint, SSL/Username, index name, index template, index state management, etc.
+
+## Pipeline Configuration
```

Comment:
I tried to keep this part consistent with the [Trace Analytics Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/trace_analytics.md), which keeps the guide direct with minimum requirements like you said. There are plenty of examples for adding SSL to all of these in their own documentation parts. Right now, I suggest we leave this section as is but adding more pipeline configurations is something we can consider before release.

Reply:
I'd like to add it, but as a separate PR. I think it would be part of #521, which we could reopen. And we would need to include it for Trace Analytics as well.

Reply:
I agree, a separate PR is appropriate.